SlugifierInterface.php

```diff
@@ -9,21 +9,31 @@
  * file that was distributed with this source code.
  */
 
+namespace Symfony\Cmf\Bundle\CoreBundle\Slugifier;
+
+if (!interface_exists('Symfony\Cmf\Bundle\CoreBundle\Slugifier\SlugifierInterface')) {
+    /**
+     * @deprecated Since 1.0, to be removed in 2.0.
+     */
+    interface SlugifierInterface {
```

Comment:
yep, "To be removed in 2.0" sounds right. if core bundle 2.0 comes later than others, the BC interface will still lay around a bit longer, that does no harm.